apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.ts

```diff
@@ -43,6 +43,7 @@ export class VoMembersComponent implements OnInit {
     Urns.USER_DEF_ORGANIZATION,
     Urns.USER_DEF_PREFERRED_MAIL,
     Urns.MEMBER_DEF_EXPIRATION,
+    'urn:perun:member:attribute-def:virt:isLifecycleAlterable',
```

Comment:
add this attribute to the `Urns` class and use it here like the other attributes